src/actions/Actions.ts

```diff
@@ -49,6 +50,8 @@ const InternalActions = {
         createAction(ActionTypes.INCREASE_HIGHEST_SEEN_POST_ID),
     addOwnFeed: (feed: PostFeed) =>
         createAction(ActionTypes.ADD_OWN_FEED, { feed }),
+    openFeed: (feedUrl: string) =>
```

Comment:
Is this used at all?

Reply:
no, I decided to go with the infamous react-navigation params. It was late and there is some leftover code 😿